### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by Daltz333**

### CODE REVIEW

**Comment by Daltz333** [comment]
Not sure why it's failing, it works locally. Any ideas @TheTripleV

**Comment by Modelmat** [comment]
> Not sure why it's failing, it works locally. Any ideas

You don't install the extension in the CI.

See 
https://github.com/Daltz333/sphinxext-opengraph/blob/efda5e6776c106d81e54596a22facdcc856753a4/.github/workflows/workflow.yml#L43-L45 for when it's done for other tests.

It will work locally because you have the extension installed locally already.

**Comment by Daltz333** [comment]
It is installed, see https://github.com/Daltz333/sphinxext-opengraph/blob/efda5e6776c106d81e54596a22facdcc856753a4/.github/workflows/workflow.yml#L61

**Comment by Modelmat** [comment]
Open graph is not in those requirements.txt files.

**Comment by Daltz333** [comment]
It is, it's ../

**Comment by ItayZiv** [comment]
You need to uncomment these lines in `conf.py`
```
# import os
# import sys
# sys.path.insert(0, os.path.abspath('.'))
```
and change the last line to 
```
sys.path.insert(0, os.path.abspath('../..'))
```

File: docs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'sphinxext-opengraph'
+copyright = '2020, FIRST'
+author = 'WPILib'
+
+# The full version, including alpha/beta/rc tags
+release = '1.0'
```
Comment (TheTripleV) [inline docs/source/conf.py:25]: Shouldn't this match the version of the package?

File: docs/source/conf.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = []
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = 'sphinx_rtd_theme'
```
Comment (TheTripleV) [inline docs/source/conf.py:52]: Can we use furo instead? I think it looks nice.

File: docs/source/conf.py (hunk 1 above)
Comment (Daltz333) [reply docs/source/conf.py:25]: Doesn't matter too much, it only shows up in the PDF version of the docs, which will be disabled

**Comment by TheTripleV** [comment]
Can you have the ci upload the built html as an artifact?

**Review by TheTripleV — APPROVED** [review]

**Pull request merged**